lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu'
+require_relative 'text'
+require 'twilio-ruby'
+# require 'dotenv'
+# Dotenv.load
+
+
+class Customer
+  include Text
+
+  attr_reader :menu
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+  end
+
+  def select_dishes(dish, quantity=1)
+    fail "This dish is not on our menu" unless menu.menu_items.key?(dish)
+    quantity.times { |order| @order << dish }
+    total_order(dish, quantity)
+  end
+
+  def total_cost
+    '£' + @order_total.to_s
+  end
+
+  def pay(amount)
+    fail "The amount given does not equal the total cost" unless amount == @order_total
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)